lecture2gether-vue/src/components/Toolbar.vue

```diff
@@ -93,8 +93,14 @@ export default class Toolbar extends Vue {
                 throw new Error(`${response.status}: Resource not available`);
             });
         }
-        // update the url to point to the lecture2go playlist when it is a
-        if (this.isValidVideoUrl(this.url) || this.url.includes('lecture2go') || this.url.includes('/l2go/')) {
```

Comment:
It looks like the check if the URL contains e.g. `lecture2go` is missing in the new version. Or is this handled in another way?

Reply:
As I can see it; yes that check has been removed. But I don't really understand what it was supposed to do?

Reply:
I made this comment a long time ago, but I think it this check checks if the entered url is a valid one for one of our services (l2f, yt, mp4, ...) otherwise the red url bar is shown.